src/background/actions/action-payloads.ts

```diff
@@ -5,7 +5,7 @@ import { BaseTelemetryData, TelemetryData, ToggleTelemetryData } from '../../com
 import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
 import { DetailsViewPivotType } from '../../common/types/details-view-pivot-type';
 import { ManualTestStatus } from '../../common/types/manual-test-status';
-import { TargetAppData, ToolData, UnifiedResult, UnifiedRule } from '../../common/types/store-data/unified-data-interface';
+import { ScreenshotData, TargetAppData, ToolData, UnifiedResult, UnifiedRule } from '../../common/types/store-data/unified-data-interface';
```

Comment:
the path of this import could be updated to:
`common/typesstore-data/unified-data-interface`

Reply:
It can't find the module when I remove the two levels up from the path